### PR TITLE
Add Meetup in footer + translation

### PR DESCRIPTION
**Pull request by nilovna**
* Change EventBrite to Meetup in footer
* Update translation

### CODE REVIEW

**Comment by mlhamel** [comment]
tumblr_o2svyahqat1rbhnqko1_400

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -127,75 +116,79 @@ msgstr "Votre messsage a été envoyé"
 
 #: mtlpy/templates/contact.html:17
 msgid "Contact us with the form below or by emailing "
-msgstr ""
+msgstr "Contactez-nous via le formulaire suivant ou par courriel"
 "Contactez-nous en utilisant le formulaire suivant ou en nous écrivant un "
 "courriel"
```
Comment (mlhamel) [inline locale/fr/LC_MESSAGES/django.po]: Érrriiiicc :-)

Comment (pior) [inline locale/fr/LC_MESSAGES/django.po]: Éric!

Comment (nilovna) [inline locale/fr/LC_MESSAGES/django.po]: ÉRICCCCC

Comment (merwok) [inline locale/fr/LC_MESSAGES/django.po]: :clap:

**Pull request merged**